jekyll/_cci2/insights-partnerships.md

```diff
@@ -17,104 +17,80 @@ providers.
+### Configure Webhooks
+{: #configure-webhooks }
+#### **Step 1. Configure Hosted Collector**
+{: #step-1-configure-hosted-collector }
 
-You will need to create one called `circleci/job-collector` and another called `circleci/workflow-collector`.
+Follow the Sumo Logic documentation for [Configuring a Hosted Collector](https://help.sumologic.com/03Send-Data/Hosted-Collectors/Configure-a-Hosted-Collector).
 
-To create the two source collectors:
-1. From the dashboard select the **Setup Wizard**.
-2. Select **Set Up Streaming Data**.
-3. Scroll to the bottom and select **All Other Sources**.
-4. Select **HTTPS Source**.
-5. For the `Source Category` enter one of the two mentioned above.
-6. Save the resulting URL.
+#### **Step 2. Add an HTTP Source**
+{: #step-2-add-an-http-source }
 
-##### 4. Add environment variables.
-{: #4-add-environment-variables }
-{:.no_toc}
+To get the URL where the CircleCI Webhooks will be sent to, and then record to the collector, we must [add an HTTP Source](https://help.sumologic.com/03Send-Data/Sources/02Sources-for-Hosted-Collectors/HTTP-Source).
 
-For each of the URLs produce in the previous step, create the corresponding environment variable.
+When complete, copy the generated “HTTP Source Address”. You can always get this link from Sumo Logic again in the future.
 
-Env vars:
-- `JOB_HTTP_SOURCE`
-- `WORKFLOW_HTTP_SOURCE`
+#### **Step 3. Configure Project Webhooks**
+{: #step-3-configure-project-webhooks }
 
-**[How to add an environment variable to your project.]({{ site.baseurl }}/2.0/env-vars/#setting-an-environment-variable-in-a-project)**
+For each project on CircleCI you wish to track, configure a webhook directed at the HTTP Source Address.
+Follow the [CircleCI docs for configuring webhooks](https://circleci.com/docs/2.0/webhooks/?section=projects#setting-up-a-hook).
 
-This will link the orb with your Sumo Logic dashboard.
+When configuring the webooks, ensure to include both the “workflow-completed”, and “job-completed” events.
 
-Your Sumo Logic dashboard will now begin to populate with data as each job runs on CircleCI.
+## Install the CircleCI App for Sumo Logic
+{: #install-the-circleci-app-for-sumo-logic }
 
+Now that you have set up collection, install the Sumo Logic App for CircleCI to use the preconfigured searches and Dashboards that provide insight into your CI Pipeline.
+To install the app:
 
-## See also
-{: #see-also }
+Locate and install the app you need from the App Catalog. If you want to see a preview of the dashboards included with the app before installing, click Preview Dashboards.
+1. From the App Catalog, search for and select the app.
+2. Select the version of the service you're using and click Add to Library. Version selection is applicable only to a few apps currently. For more information, see the [Install the Apps from the Library](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library).
```

Comment:
```suggestion
2. Select the version of the service you're using and click Add to Library. Version selection is applicable only to a few apps currently. For more information, see the [Install the Apps from the Library](https://help.sumologic.com/05Search/Library/Apps-in-Sumo-Logic/Install-Apps-from-the-Library) document.
```

Nit: I feel like the name of the linked to article doesn't quite mesh with the sentence. Your call.